feature-tests/utils/ais-events.ts

```diff
@@ -1,9 +1,27 @@
-const now = Date.now();
-const ageInSeconds = 1;
-const seconds = now / 1000 - ageInSeconds;
-const ms = seconds * 1000;
+const seconds = 12_345;
+const ms = 12_345_000;
 
-export const aisEvents = {
+interface Event {
+  timestamp: number;
```

Comment:
could we add the event_id as an optional string

Reply:
Thank you Taher, will look to add this change on a future PR, see [ATB-1404](https://govukverify.atlassian.net/browse/ATB-1404) for tracking